app/src/org/commcare/android/database/app/models/UserKeyRecord.java

```diff
@@ -54,22 +58,42 @@
     @Persisting(1)
     @MetaField(META_USERNAME)
     private String username;
+
     @Persisting(2)
     private String passwordHash;
+
     @Persisting(3)
     private byte[] encryptedKey;
+
     @Persisting(4)
     private Date validFrom;
+
     @Persisting(5)
     private Date validTo;
-    @Persisting(6)
+
     /** The unique ID of the data sandbox covered by this key **/
+    @Persisting(6)
     @MetaField(META_SANDBOX_ID)
     private String uuid;
+
     @MetaField(META_KEY_STATUS)
     @Persisting(7)
     private int type;
 
+    /** The un-hashed password wrapped by a numeric PIN **/
+    @Persisting(8)
+    private byte[] passwordWrappedByPin;
+
+    @Persisting(9)
+    private String rememberedPassword;
```

Comment:
W/o digging too much I don't understand what this is for. Is it set when using pin login and otherwise unset? A comment would help